MI-Vote/mivote-ios
# Check registration form

Test:
- Check to make sure fields that are needed in form are there
- Check to make sure submit function triggers

 Note:
- Left out verification because was unsure the format necessary for each input
